Makefile

```diff
@@ -35,7 +35,7 @@ ifdef CDP_PULL_REQUEST_NUMBER
 endif
 
 
-PATH := $(GOPATH)/bin:$(PATH)
+PATH := "$(GOPATH)/bin:$(PATH)"
```

Comment:
Why are the `"` necessary for you?

Reply:
I have some _spaced values_ in my `PATH` on Mac (`Application Support`...), and thought I'd commit it as it doesn't change existing behavior for users who don't need them.

Reply:
Ok. Fine with me then :)